app/models/case_log.rb

```diff
@@ -1,5 +1,32 @@
-class CaseLog < ApplicationRecord
-  enum status: { "in progress" => 0, "submitted" => 1 }
+class CaseLogValidator < ActiveModel::Validator
+
+  def validate_tenant_age(record, validation_regex)
+    regexp = Regexp.new validation_regex
+    if !record.tenant_age?
```

Comment:
Could you use `if record.tenant_age.blank?` here instead? I think it might be clearer than the `!`